.example.env

```diff
@@ -30,3 +30,14 @@ PINATA_PINNER_SECRET=
 
 # Github access token (for project archival requests)
 GITHUB_ACCESS_TOKEN=
+
+# Firebase stuff
+NEXT_PUBLIC_FIREBASE_API_KEY=
```

Comment:
**question** is this safe to be public?

Reply:
yes, these ones are the serverside variables:

FIREBASE_CLIENT_EMAIL=
FIREBASE_PRIVATE_KEY=

https://medium.com/@paulbreslin/is-it-safe-to-expose-your-firebase-api-key-to-the-public-7e5bd01e637b#:~:text=In%20a%20word%2C%20yes.,security%20risk%20to%20expose%20it.

Reply:
Purrrfect